bundle/manifests/rhods-operator.clusterserviceversion.yaml

```diff
@@ -1663,15 +1663,15 @@ spec:
   - llamastack
   links:
   - name: Red Hat OpenShift AI
-    url: https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.23.0
+    url: https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/3.0
```

Comment:
Have we always had the format wrong here?

Reply:
:shhhhh: